airbyte-webapp/src/packages/cloud/views/users/AccountSettingsView/AccountSettingsView.tsx

```diff
@@ -63,7 +65,7 @@ const AccountSettingsView: React.FC = () => {
         title={
           <Header>
             <FormattedMessage id="settings.accountSettings.logoutLabel" />
-            <LoadingButton danger onClick={() => logout()} data-testid="button.signout">
+            <LoadingButton danger onClick={() => logout()} isLoading={isLoggingOut} data-testid="button.signout">
```

Comment:
This one throws the error to the console:

<img width="888" alt="image" src="https://user-images.githubusercontent.com/20929344/177128474-f0f7fa30-fc53-49f0-b3c8-b82deaa15ab0.png">

Reply:
Neither CI nor I locally can reproduce this. DOes it help for you if you maybe clear your `node_modules` and install it again?